src/Registration.js

```diff
@@ -66,13 +68,26 @@ export async function startAnyRegistrationFlow(options) {
     //     });
     //} else {
         const QuestionDialog = sdk.getComponent("dialogs.QuestionDialog");
-        Modal.createTrackedDialog('Registration required', '', QuestionDialog, {
-            title: _t("Registration Required"),
-            description: _t("You need to register to do this. Would you like to register now?"),
-            button: _t("Register"),
+        const modal = Modal.createTrackedDialog('Registration required', '', QuestionDialog, {
+            hasCancelButton: false,
```

Comment:
I set this to false in an attempt to remove the cancel button as requested. However, setting this also seems to have hidden the [x] in the top right, so there is now no way to hide it other than clicking off it.

Reply:
Yeah, the two are coupled together in a QuestionDialog, so you would have to make a custom dialog to do this - would suggest we keep all our QuestionDialogs consistent and remove the cancel button for all of them as presumably the reasoning applies equally to any QuestionDialog - @nadonomy?

Reply:
I can probably give that a go if that's the way it should be done. For now I have turned the cancel back on (assuming that we still want the flag to specify the presence of the [x] and the button may go away at a future date).

Reply:
Generally speaking, we want to use _one of_ either [x] or an explicit [Cancel] button. I'm hesitant to suggest we change it generically everywhere without stepping through the user journey everywhere it's used.

Rather than risking changing it generically, could we somehow special case/pass through a prop to not show the cancel button for this specific instance of the QuestionDialog component?

Reply:
I have now implemented a switch for showing only the [x] in the corner and not the cancel button in QuestionDialog, so this can be changed case by case.